.github/workflows/security-audit.yml

```diff
@@ -0,0 +1,28 @@
+name: Security Audit
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do we really want to also trigger this on every push to main?
I think the daily schedule should be sufficient.

Reply:
we plan to have no pushes to master so often... Therefore I guess it is not such painful to have it like that

Reply:
It was needed in order to register the workflow. But I guess as we're on the default branch, it should be redundant. I'll test to remove it